opentelemetry-sdk/src/opentelemetry/sdk/_metrics/export/__init__.py

```diff
@@ -97,7 +97,7 @@ def shutdown(self) -> None:
 
 
 class InMemoryMetricReader(MetricReader):
-    """Implementation of :class:`MetricReader` that returns its metrics from :func:`metrics`.
+    """Implementation of MetricReader that returns its metrics from :func:`get_metrics`.
```

Comment:
Shouldn't we keep the `:class:`?

Reply:
This was causing an error in sphinx which is why i removed it.